recipes/conversation_view/attachment_carousel/attachment_carousel.vue

```diff
@@ -191,8 +167,19 @@ export default {
   },
 
   methods: {
+    mediaComponent (type) {
+      switch (type) {
+        case 'image':
+          return 'dt-image-carousel';
+        default:
+          return 'dt-image-carousel';
+      }
+    },
+
     removeMediaItem (index) {
-      this.$emit('remove-image', index);
+      // make sure the carousel arrows is updated. 64 is the width of each media item
+      this.showRightArrow = this.$refs.carousel.scrollWidth > (this.$refs.carousel.clientWidth + 64);
```

Comment:
Remove magic number "64". Make it a constant called MEDIA_ITEM_WIDTH or something like that.